src/components/LanguageEngagementCard/LanguageEngagementCard.tsx

```diff
@@ -0,0 +1,156 @@
+import {
+  Box,
+  Card,
+  CardContent,
+  CardMedia,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import { FC } from 'react';
+import * as React from 'react';
+import { EditInfoIcon, ProductIcon } from '../Icons';
+
+const useStyles = makeStyles(({ spacing }) => ({
+  card: {
+    maxWidth: '666px',
+    marginTop: spacing(1),
+  },
+  media: {
+    minHeight: '231px',
+    width: '176px',
+  },
+  cardBody: {
+    display: 'flex',
+  },
+  cardContent: {
+    display: 'flex',
+    flex: '1 1 auto',
+    padding: spacing(1, 2),
+  },
+  leftContent: {
+    display: 'flex',
+    flexDirection: 'column',
+    '& > *:not(:last-child)': {
+      marginBottom: spacing(1),
+    },
+  },
+  rightContent: {
+    display: 'flex',
+    flexDirection: 'column',
+    flex: '1 1 auto',
+    alignItems: 'flex-end',
+    justifyContent: 'flex-end',
+  },
+  statusLabel: {
+    marginTop: spacing(0),
+  },
+  esadDateLabel: {
+    marginTop: spacing(1),
+  },
+  product: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  editInfo: {
+    display: 'flex',
+    alignItems: 'center',
+    marginTop: 'auto',
+  },
+}));
+
+export interface LanguageEngagementCardProps {
+  id: string;
+} // This should be updated with the generated Fragment interface connected to API.
```

Comment:
It might not be a bad idea just to define the fragment here next to this file. GraphQL is meant to be queried right next to its usage so the result is tailored to exactly what is used. Since it's a fragment it's also only defining the shape and not where the data is coming from which works great.

This would leave less to do later as more TODOs could be finished now.